arch/arm/arm64/cache.S

```diff
@@ -0,0 +1,26 @@
+#include <xtf/asm_macros.h>
+
+/*
+ * flush_dcache_range(start, end)
+ * - x0(start) - start address of a region
+ * - x1(end)   - end address of a region
+ * Clobbers: x2, x3, x4
+ */
+ENTRY(flush_dcache_range)
```

Comment:
It was difficult to understand whether the function would clean or just clean & invalidate the cache. How about renaming the function to `clean_and_invalidate_dcache()`?

Reply:
Ok, I will rename it to clean_and_invalidate_dcache